README.md

```diff
@@ -1199,7 +1199,8 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   string["text"]
   ~~~
 
-* Use non-capturing groups when you don't use the captured result.
+* Use non-capturing groups when you don't use the captured result inside the
```

Comment:
Can you explain this change?

It seems to me like you could still want capturing groups even if you don't refer to them in the regular expression (i.e. `\1`, `\2`, etc.).

```ruby
def extract_date(string = "2021-11-08")
  match = /(\d\d\d\d)-(\d\d)-(\d\d)/.match(string)
  raise "invalid date" if match.nil?
  year, month, day = match.captures
  validate_date_pieces!(year, month, day)
  match.string
end
```